src/main/java/com/thebois/persistence/LoadSystem.java

```diff
@@ -0,0 +1,50 @@
+package com.thebois.persistence;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+
+/**
+ * Handles loading of saved objects in the save-file.
+ */
+public class LoadSystem extends AbstractPersistence {
+
+    private final ObjectInputStream objectInputStream;
+
+    /**
+     * Creates an instance of an input stream for the save-file.
+     *
+     * @throws IOException If the save-file is deleted during runtime.
+     */
+    public LoadSystem() throws IOException {
+
+        directoryExist();
```

Comment:
Here, you should just call `super();`

Reply:
See my response to this in the other comment